aquality-automation/aquality-tracking-ui
# Fix link to issue tracker

# PR Details

Report an Issue now should redirect to https://github.com/aquality-automation/aquality-tracking/issues

## Related Issue

#28 

## How Has This Been Tested

1. Log in 
1. Open menu under your user name
1. Click Report an Issue
1. Should be redirected to https://github.com/aquality-automation/aquality-tracking/issues

## Checklist
